python/interpret_community/common/explanation_utils.py

```diff
@@ -96,14 +99,30 @@ def _get_raw_feature_importances(importance_values, raw_to_output_feature_maps):
         if len(importance_values.shape) > 2:
             raw_importances = _multiply_sparse_matrix_3d_numpy_tensor(importance_values, raw_to_output_map.T)
         else:
-            raw_importances = raw_to_output_map.dot(importance_values.T).T
+            if _is_identity(raw_to_output_map):
```

Comment:
Is the _is_identity() function only called for sparse matrix?

Reply:
yep